bitnami/logstash/templates/headless-svc.yaml

```diff
@@ -25,5 +25,10 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.ports "context" $) | nindent 4 }}
+  {{- range $service := .Values.service.ports }}
+    - name: {{ $service.name }}
+      port: {{ $service.port }}
+      targetPort: {{ $service.targetPort }}
+      protocol: {{ $service.protocol }}
+  {{- end }}
```

Comment:
There's a problem with this implementation: it's not compatible with using templates at `.Values.service.ports` while the previous solution was.

Reply:
True, we'll be losing that option in this case... I guess we can con with the `omit` solution and work on reworking this logic in a future refactor for this chart.